### PR TITLE
Add build command tests

### PR DESCRIPTION
**Pull request by myakove**

### CODE REVIEW

File: rosa/tests/test_parse_command.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+
+CREATE_CLUSTER_CMD = "create cluster"
+
+
+@pytest.mark.parametrize(
+    "cmd,aws_region",
+    [
+        pytest.param("whoami", "aws_region"),
+        pytest.param("create cluster", "aws_region"),
```
Comment (oharan2) [inline rosa/tests/test_parse_command.py:14]:
```suggestion
        pytest.param(CREATE_CLUSTER_CMD, "aws_region"),
```

File: rosa/tests/test_parse_command.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    if expected.get("json_output"):
+        assert "-ojson" in res
+
+    if expected.get("auto_answer_yes"):
+        assert "--yes" in res
+
+    if expected.get("auto_mode"):
+        assert "--mode=auto" in res
+
+    if expected.get("region") and aws_region:
```
Comment (oharan2) [inline rosa/tests/test_parse_command.py:35]: `aws_region` is currently mandatory, maybe want to set:

```
def test_json(cmd, aws_region=None):
```

Comment (myakove) [reply rosa/tests/test_parse_command.py:35]: you can't do it since it parametrize test

**Review by rnetser — DISMISSED** [review]

**Review by omrirh — APPROVED** [review]

**Review by rnetser — APPROVED** [review]

**Pull request merged**